mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -137,37 +173,74 @@ const GraphSidebar = (props: {
     */
     const [graphUpdatedNumber, setGraphUpdatedNumber] = useState(0)
 
-    const dataSourceSheetIndex = graphParams.graphCreation.sheet_index
-    const graphOutput = props.graphDataJSON[dataSourceSheetIndex]?.graphOutput
-    const [_copyGraphCode, graphCodeCopied] = useCopyToClipboard(graphOutput?.graphGeneratedCode);
-    const [stepID, setStepID] = useState<string|undefined>(undefined);
-
-    const [loading, setLoading] = useState<boolean>(false)
+    // Log when the graph has been opened
+    useEffect(() => {
+        void props.mitoAPI.log('opened_graph');
+    }, []);
```

Comment:
Don't we get a log when the user creates a new graph as it is a step? Aka, I am not sure that we need to do this additional logging here. 

Additionally: I think we need to make sure logs actually anonymized; given that these params are nested, we need to check that our algorithm correctly loops down.